### PR TITLE
pv: add version 1.8.5

### PR DESCRIPTION
**Pull request by benoit-pierre**
Upstream: https://codeberg.org/a-j-wood/pv/.

### CODE REVIEW

**Comment by eli-schwartz** [comment]
Since pv is a CLI program whose job is to display terminal progress, I am curious what the general utility of it is as part of the wrapdb.

**Comment by benoit-pierre** [comment]
Still useful for assembling stuff (e.g. as part of an embedded distribution or program).

**Comment by benoit-pierre** [comment]
Similarly, I also use the libarchive wrap to get a tar executable for the target platform, not for the library itself.

File: subprojects/pv.wrap [hunk 1]
```diff
@@ -0,0 +1,9 @@
+[wrap-file]
+directory = pv-1.8.5
+source_url = https://codeberg.org/a-j-wood/pv/releases/download/v1.8.5/pv-1.8.5.tar.gz
+source_hash = d22948d06be06a5be37336318de540a2215be10ab0163f8cd23a20149647b780
+source_filename = pv-1.8.5.tar.gz
+patch_directory = pv
+
+[provide]
+program_names =
```
Comment (eli-schwartz) [inline subprojects/pv.wrap:9]: baffled how this works.

Comment (neheb) [reply subprojects/pv.wrap:9]: I assume the context is muon failing on this?

Comment (eli-schwartz) [reply subprojects/pv.wrap:9]: The context is rather that I don't understand the intention of defining this in a non-functional manner.

The fact that muon fails on this is interesting, but not as interesting as the question of "what was the goal here".

For example, the goal clearly wasn't to make meson work with `find_program('pv')`.

Comment (bgilbert) [reply subprojects/pv.wrap:9]: https://github.com/mesonbuild/wrapdb/pull/2344

**Pull request merged**